### ISSUE TITLE
Incorrect terminator for multiline string literal using @' or @"

### ISSUE THREAD

**Issue by mike-the-automator** [PowerShell/vscode-powershell#957]
### System Details

- Operating system name and version: Windows 10 Enterprise Build 14393.1358
- VS Code version: 1.14.0
- PowerShell extension version: 1.4.1
- Output from `$PSVersionTable`: pasted below.

```
Copy / paste the below commands into the PowerShell Integrated Terminal, and paste the output here

PS C:\Projects\SQLServerDBA\Inventory> code -v
1.14.0
b8cd5b9556d8b70ea560d35b903422363f6c5c40
PS C:\Projects\SQLServerDBA\Inventory> $pseditor.EditorServicesVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
1      4      1      0


PS C:\Projects\SQLServerDBA\Inventory> code --list-extensions --show-versions
ms-mssql.mssql@1.0.0
ms-vscode.PowerShell@1.4.1
PS C:\Projects\SQLServerDBA\Inventory> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.14393.1358
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.1358
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
### Issue Description

I am experiencing a problem with terminating multi-line string literals.  In the editor, when you open a string literal using a single quote (') or double quote (") the terminator is for the literal is automatically generated.  When you open a multi-line string literal using @' or @" (single or double quote preceded by the at sign) the editor only generates a single quote or double quote character as a terminator which will not terminate a string multi-line string literal.

I don't have time to grab a trace at the moment, but I'm happy to supply one if it would help in assessing this issue.  It seems pretty straightforward, so I'm not sure one will be necessary.

**Comment by RobDesideri** [comment]
Add some visual details:

## Expected behavior
![powershell terminal](https://user-images.githubusercontent.com/11737918/32338108-21ac9694-bff4-11e7-9a25-5a14e53b42d7.png)
This is the behavior in a normal PowerShell terminal.

## Actual behavior
![powershell integrated terminal](https://user-images.githubusercontent.com/11737918/32338180-52498866-bff4-11e7-8d68-19c60d163aa7.png)
This is the behavior in a _integrated_ PowerShell terminal.
Translated error: **Missing terminating character in the string**.

**Comment by rkeithhill** [comment]
Syntax highlighting is handled by the https://github.com/powershell/editorsyntax project.  Can you file this issue on that project?

**Comment by RobDesideri** [comment]
@rkeithhill Issue I have reported it's not about highlighting. Error is in the **integrated terminal itself**: I can't write here strings in this terminal beacause when I press the Enter key the terminal throw error `Missing terminating character in the string`

**Comment by rkeithhill** [comment]
Gotcha.

**Comment by rkeithhill** [comment]
BTW the PowerShell team is using a script from **this** repo - https://github.com/dotnet/cli/blob/master/scripts/obtain/dotnet-install.ps1

And it defaults to installing in `%LocalAppData%\Microsoft\dotnet`?  So why doesn't this script install into a location that is copasetic with global.json?

**Comment by SydneyhSmith** [comment]
Closing as this is fixed in PSReadLine because of multiline input. 
Closing as PSReadLine is now available in the PowerShell Preview extension, tracking that PSReadLine is not available in the main PowerShell Extension here #1793
